build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 // please update RELEASE_NOTES.md when you update this version
-def buildVersion = "2.2.3"
+def buildVersion = "2.2.4"
```

Comment:
I think this may have been taken https://github.com/govuk-one-login/ipv-cri-lib/pull/518/files